translations/en.yaml

```diff
@@ -365,7 +365,7 @@
 "SignatureRequestSend::ALLOW_REASSIGN": |-
   Allows signers to reassign their signature requests to other signers if set to `true`. Defaults to `false`.
 
-  **Note**: Only available for Premium plan and higher.
+  **NOTE**: Only available for Premium plan and higher.
```

Comment:
If we are making the NOTE consistent, can we make the use of a colon, it being in sentence, bolded, surrounded by caution, on a new line, underlined, etc consistent as well?

Reply:
Updated!